docs/DeveloperGuide.md

```diff
@@ -1209,20 +1284,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. Nurse's input is invalid command format.
 
     * 3a1. PatientSync shows an error message.
 
       Use case resumes at step 2.
 
-**Use case: Find patients by a tag**
+* 3b. Nurse's input contains invalid patient index.
+
+  * 3b1. PatientSync shows an error message.
+
+    Use case resumes at step 2.
+
+**Use case: UC09 - Delete tag from a patient**
+
+**Preconditions: Tag exists for the target patient**
 
 **MSS**
 
-1.  Nurse requests to list patients
-2.  PatientSync shows a list of patients
-3.  Nurse requests to find patients with a specific tag in the list
-4.  PatientSync finds patients with the tag
+1. Nurse requests to list patients.
+1. PatientSync shows a list of patients.
+1. Nurse requests to delete a tag from a specific patient in the list.
+1. PatientSync deletes a tag from the patient and displays success message.
```

Comment:
delete the specified tag(s) in the patient's tag list

Reply:
Updated!